### PR TITLE
Update manifest and other files to latest SDK

### LINKED ISSUE #8: Error, malformed driver, missing manifest.toml
if I use `docker exec -it bblfshd bblfshctl driver install --all`
I get 
```
Installing rust language driver from "docker://bblfsh/rust-driver:latest"... Error
Error, malformed driver, missing manifest.toml
```

full output:
```
Installing javascript language driver from "docker://bblfsh/javascript-driver:latest"... Done
Installing php language driver from "docker://bblfsh/php-driver:latest"... Done
Installing ruby language driver from "docker://bblfsh/ruby-driver:latest"... Done
Installing rust language driver from "docker://bblfsh/rust-driver:latest"... Error
Error, malformed driver, missing manifest.toml
Installing java language driver from "docker://bblfsh/java-driver:latest"... Done
Installing go language driver from "docker://bblfsh/go-driver:latest"... Done
Installing python language driver from "docker://bblfsh/python-driver:latest"... Done
Installing typescript language driver from "docker://bblfsh/typescript-driver:latest"... Done
Installing ocaml language driver from "docker://bblfsh/ocaml-driver:latest"... Error
Error, manifest unknown: manifest unknown
Installing bash language driver from "docker://bblfsh/bash-driver:latest"... Done

```

The same for ocaml driver: https://github.com/bblfsh/ocaml-driver/issues/8

### PR DESCRIPTION
**Pull request by juanjux**
Fixes #8 

Doesn't update anything of the native or the non-existing UAST/semantic annotations, just update the SDK, related files and manifest so it doesn't produce an error when installing with `--all`.

CI will fail since the driver obviously doesn't really work.

Signed-off-by: Juanjo Alvarez <juanjo@sourced.tech>

### CODE REVIEW

**Comment by juanjux** [comment]
Well, you can't break what never worked :)

The idea of all is to install anything and everything, but I don't dislike removing the almost non started drivers from it. What do you mean by "remove hashtag from the repository"? Removing the project from dockerhub?

**Comment by juanjux** [comment]
Update: the related issues are not happening anymore and `driver install --all` doesn't try to install the Rust or Ocaml drivers, so your concern @dennwc shouldn't be a problem, anyway this update is orthogonal to that and will come handy if someday we start working on these drivers.

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+
+[[constraint]]
+  version = "v2.0.x"
```
Comment (dennwc) [inline Gopkg.toml]: Why so ancient? :)

**Comment by juanjux** [comment]
Closing because as @dennwc said in the ocaml driver PR, merging this would actually break `--all` again so we better left this untouched until ready to really work on the driver.

**Pull request closed without merging**